app/services/analytics_events.rb

```diff
@@ -2660,13 +2660,14 @@ def multi_factor_auth_added_phone(enabled_mfa_methods_count:, **extra)
     )
   end
 
+  # @identity.idp.previous_event_name Multi-Factor Authentication: Added PIV_CAC
   # Tracks when the user has added the MFA method piv_cac to their account
   # @param [Integer] enabled_mfa_methods_count number of registered mfa methods for the user
   # @param [Boolean] in_account_creation_flow whether user is going through creation flow
   def multi_factor_auth_added_piv_cac(enabled_mfa_methods_count:, in_account_creation_flow:,
                                       **extra)
     track_event(
-      'Multi-Factor Authentication: Added PIV_CAC',
+      'Multi-Factor Authentication: Added PIV CAC',
```

Comment:
I hate to reopen pandora's box, but depending on the outcome of #9318, we may be establishing a new naming convention. Would you want to consider jumping to that new convention or holding off on a rename?

I guess my concern is whether it's worth "fixing" the name if the naming convention is going to change anyways.

Reply:
I think ill hold off on renaming tbh, since its gonna change anyways.

Reply:
#9318 is merged now if you want to go ahead and rebase / adjust.